tests/test_nitcorn.nit

```diff
@@ -0,0 +1,133 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import nitcorn
+import pthreads
+
+redef class Sys
+	var iface: String is lazy do
+		srand
+		return "localhost:{10000+20000.rand}"
```

Comment:
I assume that rand is here to avoid port collision on parallel tests. However, why not asking the OS to choose a random available port? If I remember correctly, passing 0 as a port let the OS choose